core/src/components/modal/modal.tsx

```diff
@@ -1044,10 +1047,15 @@ interface ModalOverlayOptions {
    */
   currentBreakpoint?: number;
   /**
-   * The point after which the backdrop will being
+   * The point after which the backdrop will begin
    * to fade in when using a sheet modal.
    */
   backdropBreakpoint: number;
+  /**
+   * Defines whether the backdrop should have a
+   * static opacity. (true for ionic theme only)
+   */
```

Comment:
Suggestion: Maybe the comment should mention that `staticBackdropOpacity` relies on the `--backdrop-opacity` variable.